Derpitcraft/README.md

```diff
@@ -34,21 +34,21 @@ All the following rules can be summarised simply, in one line. "Be kind to other
 
 ### Detailed Rules and Guidelines
 
-1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths/ killing players (The PvP policy is explained in **Pt.2**). It includes *harmful* pranking. Read more about the pranking policy in **Pt.3**. For a better idea on what consists of a player's base, structure, or farm, see **Pt.4**.
+1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Part 2**). It includes *harmful* pranking. Read more about the pranking policy in **Part 3**. For a better idea on what consists of a player's base, structure, or farm, see **Part 4**.
```

Comment:
They are points. If you want to make them parts, then make them all headings and make these links.